package.json

```diff
@@ -90,7 +90,8 @@
     },
     "hydrogen.provider": {
       "versions": {
-        "1.1.0": "provideHydrogen"
+        "1.1.0": "provideHydrogen",
+        "1.2.0": "provideHydrogen"
```

Comment:
Note that keeping around multiple versions is intentional. They can still point to the same place because the new version is a strict superset of the old one.